sql-db/sql-app/src/test/resources/oracle-compose-devservices.yml

```diff
@@ -0,0 +1,13 @@
+name: test-project
+services:
+  db:
+    image: ${IMAGE}
+    ports:
+      - '1521'
+    environment:
+      ORACLE_PASSWORD: quarkus
+      ORACLE_DATABASE: quarkus
+      APP_USER: quarkus
+      APP_USER_PASSWORD: quarkus
+    labels:
+      io.quarkus.devservices.compose.wait_for.logs: .*DATABASE IS READY TO USE.*
```

Comment:
This had to be added to make `DevModeOracleComposeIT` work, upstream documentation will need to be updated ...